index.d.ts

```diff
@@ -8,12 +8,15 @@ declare global {
     [key: string]: string
   }
 
+  type EventType = "selectionchange" | "currentpagechange" | "close" | "timerstart" | "timerstop" | "timerpause" | "timerresume" | "timeradjust" | "timerdone"
+
   interface PluginAPI {
     readonly apiVersion: "1.0.0"
     readonly command: string
 
     readonly fileKey: string | undefined
 
+    readonly timer: TimerAPI
```

Comment:
should this be timer?

Reply:
erm, can you elaborate?

Reply:
omg re-reading my comment 🤦🏻 

I meant to say... should this be `timer?` ?

Reply:
ohhh haha thank you! yes